lib/BackgroundJob/Tasks/CreateClustersTask.php

```diff
@@ -204,6 +211,8 @@ private function getCurrentClusters(array $faces): array {
 	private function getNewClusters(array $faces): array {
 		// Create edges for chinese whispers
 		$euclidean = new Euclidean();
+		$sensitivity = floatval($this->config->getAppValue('facerecognition', 'sensitivity', '0.5'));
```

Comment:
Just because it is in the middle of the scale .. :sweat_smile: 

0.4 <= sensitivity <= 0.6 

I guess it does not make sense to go beyond those values. . No?
Then it will be the administrator's choice ..